src/settingsmanager.h

```diff
@@ -25,15 +25,17 @@ class SettingsManager : public QObject
     int getKiwixServerPort() const { return m_kiwixServerPort; };
     QString getKiwixServerIpAddress() const { return m_kiwixServerIpAddress; };
     qreal getZoomFactor() const { return m_zoomFactor; };
-    QString getDownloadDir() const { return m_downloadDir; }
-    QString getMonitorDir() const { return m_monitorDir; }
+    QString getDownloadDir() const { return m_downloadDir; };
+    QString getMonitorDir() const { return m_monitorDir; };
```

Comment:
Instead of adding semicolons on the end-of-line, you'd better removed such superfluous punctuation on the other neighbouring lines